test/attributes_test.rb

```diff
@@ -53,7 +53,8 @@
 {#{name}}
         EOS
         result = convert_string_to_embedded str
-        assert_includes result, %(<p>#{value}</p>)
+        # NOTE truffleruby adds \n to the beginning of a paragraph containing RTL text
+        assert_includes result, value
```

Comment:
Again, is this something we should be fixing in TruffleRuby and Is there an associated issue?

Reply:
I'm going to try to figure out what the root cause is. Then I'll report.

Reply:
See https://github.com/oracle/truffleruby/issues/1543